modulefiles/workflow_utils.hera

```diff
@@ -20,10 +20,9 @@ module load sp/2.3.3
 module load ip/3.3.3
 module load nemsio/2.5.2
 module load nemsiogfs/2.5.3
-#module load ncio/1.0.0
+module load hdf5/1.10.6
+module load netcdf/4.7.4
```

Comment:
please move this right after `png`.
One of the `w3` libraries used to have netcdf dependency (or may be still does).

Reply:
Done